Packs/AzureSentinel/Scripts/MicrosoftSentinelSetOwnerButton/README.md

```diff
@@ -0,0 +1,30 @@
+This script is used to set owner in remote system.
+
+## Script Data
+
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | python3 |
+| Tags | dynamic-section |
+| Cortex XSOAR Version | 5.5.0 |
+
+## Inputs
+
+---
+
+| **Argument Name** | **Description** |
+| --- | --- |
+| owner_email | The owner email to set as user principal name. |
```

Comment:
is email the same as user principal name?

Reply:
Yes 
its the same one